.github/CODEOWNERS

```diff
@@ -14,4 +14,4 @@
 # limitations under the License.
 #
 
-* @polaris-catalog/polaris
+* @jbonofre @anoopj @RussellSpitzer @adutra @laurentgo @snazy @hivivo @flyrain @dennishuo @takidau @aihuaxu @jackye1995 @collado-mike @dimas-b
```

Comment:
In preparation for ASF move, we should have only the PPMC (listed in the proposal). It means `@jbonofre @ashvina @anoopj @RussellSpitzer @snazy @vvcephei @hivivo @takidau @jackye1995`

See: https://cwiki.apache.org/confluence/display/INCUBATOR/PolarisProposal

Reply:
Roger that -- thank you for the clarification! Will update

Reply:
I think it'll be important to have me and/or @dennishuo in here, at least very soon after the move. I think we have the deepest knowledge of the code base and ought to be involved with reviewing most of the changes to the actual business logic

Reply:
I think it'll be important to have me and/or @dennishuo in here, at least very soon after the move. I think we have the deepest knowledge of the code base and ought to be involved with reviewing most of the changes to the actual business logic

Reply:
Resolved this offline -- we will add committers publicly based on ASF processes like https://community.apache.org/contributors/becomingacommitter.html